app/_scss/header.scss

```diff
@@ -0,0 +1,30 @@
+// Header
+// --------------------------------------------------
+
+@import "common/vars";
+
+.Header {
+  max-width: $max-w;
+  margin: 0 auto;
+  padding: 0 10px;
+  height: 100px;
+}
+
+.Header .Logo {
```

Comment:
Merge rule `.Header .Logo` with rule on line 6<br>Selector `Header` should be written in lowercase with hyphens<br>Selector `Logo` should be written in lowercase with hyphens